fusor-ember-cli/app/components/pf-modal-base.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  openCloseModal: Ember.observer('openModal', function() {
+    if (this.get('openModal')) {
+      Ember.$('#'+this.get('idModal')).modal('show');
+    } else {
+      Ember.$('#'+this.get('idModal')).modal('hide');
+    }
+  }),
+
+  actions: {
+    closeModal: function() {
+      this.set('openModal', false);
```

Comment:
@eriknelson, good point. I removed the explicit `return`. I was influenced by ruby where the return is optional.  I see that for JS it's best to omit it and return `undefined`  and doesn't bubble.